File: 03- Subprogramas y recursividad/subprogramas/python/test_paquete.py
import paquete.operaciones_aritmeticas

def main():
    serie = 0
    n = 2

    serie = paquete.operaciones_aritmeticas.calcular_serie(n)
    print(f"El resultado de la serie es: {serie}")

# Ejecuta la aplicación
if __name__ == '__main__':
    main()

File: 03- Subprogramas y recursividad/subprogramas/java/TestPaquete.java
import javax.swing.JOptionPane;
import paquete.OperacionesAritmeticas;

public class TestPaquete {
    public static void main(String[] args) {
        double serie = 0;
        int n = 2;

        serie = OperacionesAritmeticas.calcularSerie(n);
        JOptionPane.showMessageDialog(null, "El resultado de la serie es: " + serie);
    }
}